pkg/crd/markers/validation.go

```diff
@@ -310,3 +338,13 @@ func (m Default) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
 	schema.Default = &v1beta1.JSON{Raw: marshalledDefault}
 	return nil
 }
+
+func (m XPreserveUnknownFields) ApplyToSchema(schema *v1beta1.JSONSchemaProps) error {
+	schema.XPreserveUnknownFields = pointer.BoolPtr(true)
```

Comment:
```suggestion
        defTrue := true
	schema.XPreserveUnknownFields = &defTrue
```

This is the silliest library, and I'd rather not have to deal with something else in it conflicting with a dependency.